mne/io/base.py

```diff
@@ -677,6 +677,74 @@ def set_annotations(self, annotations, emit_warning=True):
 
         This setter checks if they are inside the data range.
 
+        ----------- meas_date=XX, orig_time=YY -----------------------------
```

Comment:
Shall we keep this here? or move it to the tests?

Is there a manner to convert this into a graph in the doctests that renders properly? 

Should it go into Notes?

Reply:
Definitely in Notes. I would put it in a literal block so it renders in a fixed-width font, like:

```
blah blah blah.

::

    ascii art
    more ascii art

blah blah
```
I'd put it in the `Notes` of the `Annotation` class itself